docs-requirements.txt

```diff
@@ -3,3 +3,4 @@
 -r test-requirements.txt
 sphinx~=1.4.9
 sphinx-argparse~=0.1.15
+sphinx_rtd_theme
```

Comment:
We don't use sphinx_rtd_theme. We use `alabaster` theme which is already included when you do `pip3 install -r docs-requirements.txt`. No additional entry is needed.

@satwikkansal, Please justify why you think this is needed, otherwise, remove it :)
Thanks.